### DISCUSSION TITLE
Stolen Code?

### DISCUSSION THREAD

**Discussion by alockbox** [vishnuraghavb/EnBizCard#48] [General]
I don't know enough about this stuff, but isn't this site just using slightly modified EnBizCard code with no reference to this source? https://www.smartvcard.com

**Comment by vishnuraghavb** [comment]
This issue has been sorted now. SmartVCard just updated their source and has given proper credit to the EnBizCard project.

**Comment by alockbox** [comment]
You don’t mind that they are charging money to profit off your code? I would rather pay donate to you, if yours had the ability to save and reuse / update cards. Many people seem to use your code base on their sites.

**Reply by vishnuraghavb** [reply]
Not at all—my goal with using the AGPL license is to ensure that any modifications to the code remain open and available to everyone. The AGPL allows others to use, host, or even sell services around the app, as long as they respect the license terms and contribute back any code changes. This way, the community benefits from improvements, and everyone has a fair opportunity to innovate on the same foundation. :)

**Reply by vishnuraghavb** [reply]
> Many people seem to use your code base on their sites.

That's interesting! I would love to know who else is using it. Could you share a list or any examples of the sites you've seen using the code? It would be really helpful for me to get a better idea of its impact.

**Reply by vishnuraghavb** [reply]
The save and reuse feature is in development! I've been working on a major update but had to pause due to time constraints. I’m committed to getting it launched as soon as I can, though I don’t have an exact ETA at the moment. Thank you for your patience and support!

**Reply by vishnuraghavb** [reply]
I really appreciate your interest and support! If you’d like to help speed up development and motivate me to add new features faster, consider making a donation. Every contribution, no matter how small, encourages me to dedicate more time to improving the project. Thank you!

Donate here: https://www.vishnuraghav.com/donate

**Reply by alockbox** [reply]
> Not at all—my goal with using the AGPL license is to ensure that any modifications to the code remain open and available to everyone. The AGPL allows others to use, host, or even sell services around the app, as long as they respect the license terms and contribute back any code changes. This way, the community benefits from improvements, and everyone has a fair opportunity to innovate on the same foundation. :)

Yes but they did not contribute code back right? They altered the code, and used it without attribution, for their personal profit…

You are far more generous to benefit of the doubt than I would be!

**Reply by alockbox** [reply]
> > Many people seem to use your code base on their sites.
> 
> That's interesting! I would love to know who else is using it. Could you share a list or any examples of the sites you've seen using the code? It would be really helpful for me to get a better idea of its impact.

Links to your project are all over ever self-hosted business card conversation online! The biggest issue people experience is the having to go back and rebuild the entire card from scratch for each person for every change. It's better than than paying $9 a month per card to subscriptions while they also collect all your information and traffic stats, but it certainly would be a huge enhancement to be able to host and alter cards on the fly. It would also be great to allow custom icons uploaded (ie the Twitter one needs to be X, or just making them all the same color since insta is an odd white one, etc.).

**Reply by alockbox** [reply]
> I really appreciate your interest and support! If you’d like to help speed up development and motivate me to add new features faster, consider making a donation. Every contribution, no matter how small, encourages me to dedicate more time to improving the project. Thank you!
> 
> Donate here: https://www.vishnuraghav.com/donate

I have and will again! I would happily donate again as well for the ability to alter cards without fully rebuilding! I don’t believe I’m the only one. I think many people began to think you had given up on this project.

I tried the fork which allows a quasi-update feature but it is a little buggy and fails to update certain things correctly like featured items images, etc.

I’m not a coder, I’m a long-time hardware guy, but I would be happy to beta test any coming version you have on docker in my NAS.
